sympy/codegen/cfunctions.py

```diff
@@ -160,7 +160,7 @@ def _eval_is_zero(self):
     def _eval_is_nonnegative(self):
         return self.args[0].is_nonnegative
 
-_Two = S(2)
+_Two = Integer(2)
```

Comment:
Why do we even need `_Two` at all in this file. Everywhere where it is used can be replaced with 2 as far as I can tell, because every function will automatically sympify its input. In fact the only benefit of doing this is if we instead wrote things like `2**x` instead of `Pow(_Two, x)`. Doing both is redundant. Alternatively, functions like `_exp2` could sympify their arguments if they could receive non-SymPy inputs.

Reply:
I think the idea is to save additional sympifying/object creation, but not sure. Will add to the import time, but the be "free". In this context (codegen), it may not be worth it though.